### ISSUE TITLE
Giant files cause jigsaw error

### ISSUE THREAD

**Issue by keflavich** [CARTAvis/carta#162]
When loading very big files - or even medium-big files, like the JWST NIRCAM images - I often get a "jigsaw" failure in which the screen looks like this:

<img width="824" alt="image" src="https://user-images.githubusercontent.com/143715/187045302-1ef3ed75-4d94-481e-9694-d397b0646a0e.png">

Here's a movie showing what happens when I zoom in:
https://user-images.githubusercontent.com/143715/187045336-00b19edc-ee22-4413-864f-e51a9a39b806.mp4

In this example, the file loaded is a 2.3 GB single image (28000 x 10000 pixels)

This is with CARTA beta 3.0.0-beta3

**Comment by keflavich** [comment]
...ok apparently that version of screen capture doesn't work.  Dropbox: https://www.dropbox.com/s/0dwqgfeoi8hvsgg/jigsaw.mov?dl=0

**Comment by kswang1029** [comment]
are you using CARTA remotely? Could you load the image again and watch the top right corner of the GUI for a green cloud as an indicator of data streaming? Before the green cloud disappears, do not zoom/pan your image.

**Comment by keflavich** [comment]
Yes, I'm accessing remotely, but I'm not doing any mouse motion at all to trigger this - it happens if I open several images in a row that are position-matched and large enough.  I can screencap that later.  

The problem is, if this happens, it's unrecoverable - the artifacts (mis-located subframes) stay there permanently

**Comment by kswang1029** [comment]
One more question: do you enable image matching on append?

**Comment by keflavich** [comment]
Yes, I enable image matching on append.
